armi/operators/settingsValidation.py

```diff
@@ -203,6 +203,11 @@ def run(self, cs=None):
                 query.resolve()
                 if query.corrected:
                     correctionsMade = True
+                if correctionsMade:
+                    runLog.extra(
+                        f"At least one setting correction detected. Overwriting settings file `{self.cs.path}`."
```

Comment:
can we rename the old settings file as `fileName_old.yaml` or something like that so we retain the old file?

probably also check if that path exists and avoid overwitting something that alread exists if `fileName_old.yaml` exists

Reply:
Just updated the code to handle this better I think. Going to fix the tests now.